LandSandBoat/server
# Various Bug Fixes for Nyzul Isle Investigation

**_I affirm:_**

## What does this pull request do?
Corrects some minor oversights in the current implementation of Nyzul Isle Invesigation.

- Some NQ mobTypes are set to '16' which causes them to check as notorious monsters.
- Corrects a check that will otherwise never be triggered and thus never set a specified NM.
- Adds logic to add CHECK_AS_NM mobMod when the Kill specified group objective is selected.
- Corrects the enums for items that are rewarded from Treasure Chests



## Steps to test these changes
You can set your desired test objective here: (2 or 4)
https://github.com/LandSandBoat/server/blob/base/scripts/zones/Nyzul_Isle/instances/nyzul_isle_investigation.lua#L45

!addkeyitem 878
Enter Nyzul
Run around and see that only the correct mob, or group check as NM, and successfully trigger the Rune of Transfer when defeated.
